.drone/drone.jsonnet

```diff
@@ -653,8 +666,6 @@ local manifest_ecr(apps, archs) = pipeline('manifest-ecr') {
           RELEASE_TAG_REGEXP: '^([0-9]+\\.[0-9]+\\.[0-9]+)$',
         },
         commands: [
-          'apk add --no-cache bash git',
```

Comment:
i will cause an issue that ./tools/image-tag won't be executed becuse it uses bash